frontend/src/views/web/auth/login.html

```diff
@@ -20,8 +20,11 @@
         </div>
         <!-- password -->
         <div class="input-field align-left">
-            <input type="password" id="password" onpaste="return false;" class="dark-autofill" name="password" ng-model="auth.getUser.password" autocomplete="new-password" ng-minlength="8" required>
-            <span class="form-icon form-icon-dark"><i class="fa fa-key"></i></span>
+            <input type="{{canShowPassword ? 'text' : 'password'}}" id="password" onpaste="return false;" class="dark-autofill" name="password" ng-model="auth.getUser.password" autocomplete="new-password" ng-minlength="8" required>
```

Comment:
Please revert this change since this is not related to oauth integration. Please feel free to open a new PR for this.